basic/src/test/java/org/jboss/hal/testsuite/test/runtime/WebMetricsTestCase.java

```diff
@@ -70,9 +70,9 @@ public void before() throws IOException, TimeoutException, InterruptedException
         statementContext = new DefaultContext();
         ResourceAddress addressStats = ADDRESS_TEMPLATE_STATISTICS.resolve(statementContext);
         dispatcher.execute(new Operation.Builder(ModelDescriptionConstants.WRITE_ATTRIBUTE_OPERATION, addressStats).param("name", "statistics-enabled").param("value", "true").build());
-        cliClient.reload();
 
         if (ConfigUtils.isDomain()) {
+            ServerManager.getInstance(cliClient).reloadAndWaitUntilRunning(60000);
```

Comment:
In this case initializing directly `DomainManager` would make more sense, as the `reloadAndWaitUntilRunning` was added to standalone even though not needed as `CliClient.reload` automatically waits for server to start and in standalone mode there is no confusion with waiting only for host vs all servers as in domain.